sirbot_pyslackers/endpoints/slack/messages.py

```diff
@@ -42,14 +49,21 @@ async def stock_quote(message, app):
     if not match:
         return
 
-    asset_class, symbol = match.group("asset_class"), match.group("symbol")
+    asset_class, symbol, currency = (
+        match.group("asset_class"),
+        match.group("symbol"),
+        match.group("currency"),
+    )
+    if not currency or not CRYPTO_CURRENCIES.get(currency, False):
```

Comment:
You can simplify this a decent bit: 

```python
quote_currency = CRYPTO_CURRENCIES.get(match.group("currency"), "USD")
```

There's no reason to check for existence since the empty value if the `currency` group is not found in the message is `None` which will not be found in the `CRYPTO_CURRENCIES` dict falling back to `"USD"`

Reply:
`CRYPTO_CURRENCIES.get(match.group("currency"), "USD")` would return the symbol, not the currency code though. I actually did exactly this originally, then realised my mistake :) The `get` and default value would need to apply to the `match.group`, and it doesn't support `get` like that (I double checked).

Reply:
I could do 
```
currency = match.group("currency") or "USD"
```
That would be neater and should work I think, since match.group("currency") would be `None` if not specified, but I still need to verify it against the dict. Or. 
``` 
currency = match.group("currency") if match.group("currency") in FIAT_CURRENCY.keys() else "USD"
```

Reply:
Not sure if I should be resolving these or you? I'm sure you can unresolve them if you want to though :)

Used `currency = match.group("currency") if match.group("currency") in FIAT_CURRENCY.keys() else "USD"` for a one-liner.

Reply:
I missed the intent here, good call :D